.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
 .github @lidofinance/review-gh-workflows
+* @lidofinance/lido-ui-core
```

Comment:
is syntax like "!.github" supported here?

Reply:
@Jabher yep, the same syntax as in gitignore, but I don't want to exclude .github folder, feels like it still makes sense for owners to review it